workflows/ncn/hooks/before-all/install-csi.yaml

```diff
@@ -31,6 +31,28 @@ spec:
   scriptContent: |
     #!/bin/sh
     source /srv/cray/scripts/metal/metal-lib.sh
+
```

Comment:
Adds wait condition for nexus, which is singleton may be migrated at the same time in parallel step.